django_prices/templatetags/prices.py

```diff
@@ -13,24 +14,13 @@ def normalize_price(price, normalize):
     return price
 
 
-@register.inclusion_tag('prices/price.html')
-def gross(price, normalize=False):
-    return {'amount': normalize_price(price.gross, normalize),
-            'currency': price.currency}
-
-
-@register.inclusion_tag('prices/price.html')
-def net(price, normalize=False):
-    return {'amount': normalize_price(price.net, normalize),
-            'currency': price.currency}
-
-
-@register.inclusion_tag('prices/price.html')
-def tax(price, normalize=False):
-    return {'amount': normalize_price(price.tax, normalize),
-            'currency': price.currency}
+@register.filter
+def amount(obj, normalize=False):
```

Comment:
Maybe `render_amount` or something similar would be a better name.

Reply:
@patrys what do you think?

Reply:
I prefer `amount` as it will only exist in templates where pretty much everything is about rendering stuff 😉

Reply:
👍